src/app/containers/Metadata/index.jsx

```diff
@@ -61,6 +62,20 @@ const MetadataContainer = ({ metadata, promo }) => {
                 />
                 <Metadata
                   isAmp={platform === 'amp'}
+                  alternateLinks={[
```

Comment:
This sets the alternate links to be `en` and `en-gb` for all services. A later PR will be needed to generalise this for all non-English services. This array of values will then need to live in the `ServiceContext`.